public/css/genericStyles.css

```diff
@@ -43,7 +43,7 @@ input[type="submit"] {
     border-top: 1px solid #CCCCCC
 }
 
-.moreInfo {
+.more-info {
```

Comment:
I think this was done in Matt's branch but margin-top and margin-bottom doesn't have to be in two separate lines. 
margin: 10px 0;
should do the job.